packages/messaging/src/ui/components/ContentCardContainer/ContentCardContainer.tsx

```diff
@@ -23,10 +23,13 @@ export interface ContentCardContainerProps<T> extends Partial<FlatListProps<T>>
   FallbackComponent?: ReactElement | null;
   EmptyComponent?: ReactElement | null;
   surface: string;
+  settings: ContainerSettings | null;
+  isLoading?: boolean;
```

Comment:
because we moved the hook out of this component, we should also these state pieces and their components